### PR TITLE
Medical info page

### PR DESCRIPTION
**Pull request by Gtstride**

### CODE REVIEW

File: backend/config/keys_dev.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  mongoURI:
+    // 'mongodb+srv://godstime:eWC2hZqacjFP8KZH@cluster0-7achi.mongodb.net/test?retryWrites=true&w=majority',
+    "mongodb+srv://godstime:9OTpPdtiwoL7uEMQ@cluster0-rorws.mongodb.net/test?retryWrites=true&w=majority",
```
Comment (robinv85) [inline backend/config/keys_dev.js:4]: this should be in `.gitignore` ? or is this for everyone to use mongo atlas in the cloud

File: client/src/components/Logo/index.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
 import styled from "styled-components";
 
 const Logo = styled.img`
+<<<<<<< HEAD
```
Comment (robinv85) [inline client/src/components/Logo/index.js:4]: merge conflicts?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fightpandemics",
```
Comment (robinv85) [inline package.json:2]: not sure why you added package.json in root? this was probably moved, did you start from an old outdated branch and tried to put it back  on master?

File: client/src/styles/colors.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 $white: #fff;
```
Comment (anniegiang) [inline client/src/styles/colors.scss:1]: @robinv85 I think we'll be removing `.scss` files right?

Comment (robinv85) [reply client/src/styles/colors.scss:1]: yeah colors should go in theme constants now, but something like "white" may already be in there?

File: package.json (hunk 3 above)
Comment (Gtstride) [reply package.json:2]: No, not at all, before I start off on any of the tasks, I make it a point of duty to update my local branch with the remote master branch.

File: client/src/styles/colors.scss (hunk 4 above)
Comment (Gtstride) [reply client/src/styles/colors.scss:1]: Okay, point noted, every color goes into the constant files henceforth

File: client/src/components/Logo/index.js [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import styled from "styled-components";
 
 const Logo = styled.img`
+  width: 210px;
```
Comment (anniegiang) [inline client/src/components/Logo/index.js:4]: Can we use `rem` values for these?  Right now 1rem = 10px.

File: client/src/common/colors.js [hunk 6]
```diff
@@ -0,0 +1 @@
+export const PRIMARY = "#6A7EFC";
```
Comment (robinv85) [inline client/src/common/colors.js:1]: hmm how come this is in `/src/common`? I thought this should be `constants/colors.js` ... folder was renamed like 2 weeks ago or so?

**Comment by anniegiang** [comment]
@Gtstride Please resolve the merge conflicts.

**Comment by robinv85** [comment]
@Gtstride this was merged in another PR and can be closed, right?

File: client/src/common/colors.js (hunk 6 above)
Comment (Gtstride) [reply client/src/common/colors.js:1]: i had to pull and rebase, my locals were not up to date

File: client/src/components/Logo/index.js (hunk 5 above)
Comment (Gtstride) [reply client/src/components/Logo/index.js:4]: okay

File: client/src/components/Logo/index.js (hunk 2 above)
Comment (Gtstride) [reply client/src/components/Logo/index.js]: This has been resolved.

**Pull request closed without merging**